db/seeds.rb

```diff
@@ -79,6 +78,60 @@
   end
   puts "Completed seeding standup meeting group!"
 
+
+  puts "Seeding a user_mentee_application..."
+
+  user_mentee_application_data = [
```

Comment:
we can do this separately, but one thing we should do is allow factorybot to be used in seed files to really clean this up.